signer/signer_test.go

```diff
@@ -303,3 +310,190 @@ var rejectedFileNames = []string{
 	"control*chars",
 	"_non_alpha_start",
 }
+
+func MockedPKCS11ContextFactory(ctx crypto11.PKCS11Context) crypto11.PKCS11ContextFactory {
```

Comment:
this could be lower case. When I saw it elsewhere, I went looking to see where this was being defined and then used outside of this package and these tests.